lib/Service/Note.php

```diff
@@ -105,6 +105,9 @@ public function getData(array $exclude = []) : array {
 		if (!in_array('readonly', $exclude)) {
 			$data['readonly'] = $this->getReadOnly();
 		}
+		$data['internalPath'] = $this->file->getInternalPath();
```

Comment:
I pushed a small fix for getting the proper relative path for the user to be used in the frontend.